src/mlpack/methods/reinforcement_learning/environment/mountain_car.hpp

```diff
@@ -0,0 +1,172 @@
+/**
+ * @file mountain_car.hpp
+ * @author Shangtong Zhang
+ *
+ * This file is an implementation of Mountain Car task
+ * https://gym.openai.com/envs/MountainCar-v0
+ *
+ * TODO: refactor to OpenAI interface
```

Comment:
I'm not sure that we should fully compliant with the OpenAI Interface. What do you think?